packages/server/src/mail/user-subscriptions.entity.ts

```diff
@@ -0,0 +1,30 @@
+import {
+  Entity,
+  PrimaryGeneratedColumn,
+  ManyToOne,
+  BaseEntity,
+  Column,
+} from 'typeorm';
+import { UserModel } from '../profile/user.entity';
+import { MailServiceModel } from './mail-services.entity';
+
+@Entity('user_subscriptions')
+export class UserSubscriptionModel extends BaseEntity {
+  @PrimaryGeneratedColumn()
+  id: number;
```

Comment:
As a little tip, if you're not using this anywhere (I haven't looked at the code closely enough to see if you do), don't use a PrimaryGeneratedColumn for a many-to-many relationship.

There's a couple big reasons for this:
- Having a PrimaryGeneratedColumn essentially means that this entity can exist on its own (e.g. the user and/or service is set to null). This doesn't make sense as you probably don't want users subscribed to a null service or services with null users. 
- If you are not querying on this id, then it will hurt performance rather than gain performance. And usually with these many-to-many tables, you're querying on the userId and the serviceId rather than this id. So unless you specifically query on this PrimaryGeneratedColumn, remove it.

What you're probably looking for is to make this a partial primary key. studentTaskProgress.entity has an example of how to do this (with two @PrimaryColumn() and with two @JoinColumn)

Reply:
Good point, I deleted generated id. 
Regarding @ joincolumn, that's what I had, however, I need to actively query the join table and thus I created this join table explicitly.